install_sct

```diff
@@ -537,6 +537,10 @@ else
   # Not a package
   print info "Using requirements.txt (git installation)"
   pip install -r requirements.txt
+  yes | pip uninstall tensorflow-tensorboard
```

Comment:
```suggestion
  # `tensorflow-tensorboard` is installed by `tensorflow==1.5.0` but is not needed, 
  # and conflicts with the other installation of `tensorboard`. So, we uninstall here.
  # This is hacky, and should be removed as soon as we stop using `tensorflow==1.5.0`.
  # See https://github.com/neuropoly/spinalcordtoolbox/issues/3035 for more info.
  yes | pip uninstall tensorflow-tensorboard
```

Reply:
I agree that this is hacky. Are we doing this to avoid telling the user to install `libomp`?

Reply:
No we are doing this to use ivadomed 2.5 which is needed in #3041 for example.

Reply:
Definitely cleaner to remove the `tensorflow==1.5.0` dependency and update code to use the latest version. Is there a reason we need `1.5.0`?

Alternatively I have seen somewhere that it is also possible to do:
```python
# in SCT before tensorflow is ever imported
import pkg_resources
pkg_resources.require("tensorflow==1.5.0")
import tensorflow as tf15 # import tensorflow could also probably work? TBD

tf15.do_something()
```
Which should avoid the namespace clash with `ivadomed`.

To be honest I think it's better to update the code unless we have a strong incentive to maintain compatibility with `tensorflow==1.5.0`. It just makes our lives harder.

Reply:
We pin `tensorflow==1.5.0` to support older CPUs that do not support AVX, see #2618. More info about the `tensorboard` clash in #3035.

The trouble with waiting until we move away from `tensorflow` is that it's quite a large effort (I think), see #3046. So it may take some time, and cause us to fall out of sync with `ivadomed`'s development.

Reply:
Yes there is a reason explained in #2618. We will eventually get rid of it but it will take some time and might prevent current development.

Reply:
Ah ok I understand. This is because `pip` fetches a pre-built version of tensorflow that is compiled with the AVX instruction set. Computers that don't have this instruction available should use a version that was built without the `AVX` compilation flag. 

For reference: https://github.com/tensorflow/tensorflow/issues/24548

Looking at the wiki for AVX it appears to have been introduced a while ago (2011): https://en.wikipedia.org/wiki/Advanced_Vector_Extensions which may explain why `tensorflow` don't even bother with the non-AVX binary.

In any case it is definitely possible to compile a recent version of `tensorflow` without the AVX CPU flag. Question is do we want to bother with that (or put the burden on the user, which I already know @jcohenadad is opposed to :smile: )

Maybe we can find some pre-compiled binaries online for all platforms.